templates/pages/core/get-started/installation-medias.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Create installation medias for Ubuntu Core
+description: These steps will walk you through creating a bootable Ubuntu Core SD Card or USB flash drive.
+---
+
+# Create installation medias for Ubuntu Core
+
+These steps will walk you through creating a bootable Ubuntu Core SD Card or USB flash drive.
+
+This will erase any existing content on the removable drive.
+
+The process is the same on any OS, you will: download an image file, insert your removable drive (SD Card, USB flash drive, etc.), unmount it, replace the drive content with the image file content, eject the drive.
+
+You can follow the detailed steps:
```

Comment:
Delete this section as we will be generating the anchorlink contents from the title headings